### PR TITLE
Updated code & cordova ios version

### PR DESCRIPTION
**Pull request by RiyaMRoy04**
- Updated Dockerfile
- Updated from watson-developer-cloud npm package to ibm-watson
- Updated cordova ios version since the earlier version was throwing an error.

### CODE REVIEW

File: server/app.js [hunk 1]
```diff
@@ -5,8 +5,8 @@ const application = express();
 const fs = require("fs");
 const formidable = require("formidable");
 const tesseract = require("node-tesseract");
-const LanguageTranslatorV3 = require("watson-developer-cloud/language-translator/v3");
-const NaturalLanguageUnderstandingV1 = require('watson-developer-cloud/natural-language-understanding/v1.js');
```
Comment (stevemar) [inline server/app.js:9]: Hmm, i'm not sure if the version parameter is OK to pass through, maybe it's just ignored?

https://github.com/watson-developer-cloud/node-sdk/releases

```
natural-language-understanding: The version_date parameter in Natural Language Understanding is no longer supported.
```

See line 17 below

**Review by stevemar — COMMENTED** [review]
rest of the changes look good, just one question/comment.

File: server/package.json [hunk 2]
```diff
@@ -7,7 +7,7 @@
     "express": "^4.16.3",
     "formidable": "*",
     "node-tesseract": "*",
-    "watson-developer-cloud": "3.8.0"
+    "ibm-watson": "*"
```
Comment (sanjeevghimire) [inline server/package.json]: I think we should also pin the `ibm-watson` version rather than getting latest.

File: server/app.js (hunk 1 above)
Comment (RiyaMRoy04) [reply server/app.js:9]: Hey @stevemart, It is mentioned that **version_date** is no longer supported but use **version** instead to migrate. 
Screenshot 2019-05-03 at 12 03 45 PM

I have used the **version** parameter in the code.
```
let natural_language_understanding = new NaturalLanguageUnderstandingV1({
  version: '2018-03-16'
});
```

File: server/package.json (hunk 2 above)
Comment (RiyaMRoy04) [reply server/package.json]: @sanjeevghimire  Sure, I have updated the version to 4.1.0 now.

**Comment by stevemar** [comment]
Thanks Riya!

**Review by stevemar — APPROVED** [review]

**Pull request merged**